INSTALLING.md

```diff
@@ -0,0 +1,78 @@
+> ⚠️ Avoid using either system-installed Python or Python from Homebrew, as these versions are often labeled `--externally-managed` resulting in restricted dependency installation.
```

Comment:
I think also previous chapter from README can be moved here. It should not be Apple-Silicon only document